toqito/channel_metrics/completely_bounded_spectral_norm.py

```diff
@@ -7,7 +7,7 @@
 
 def completely_bounded_spectral_norm(phi: np.ndarray) -> float:
     r"""Compute the completely bounded spectral norm of a quantum channel.
-    
+
     :cite:`Watrous_2009_Semidefinite, QETLAB_link`.
```

Comment:
I wonder if a citation on its own here will look awkward when rendered. As I don't think the QETLAB link makes sense here, perhaps just:

```suggestion
r"""Compute the completely bounded spectral norm of a quantum channel :cite:`Watrous_2009_Semidefinite`.
```